test/memote.jl

```diff
@@ -70,3 +70,22 @@ end
 
     @test length(reactions_transport_no_gpr(model; config = memote_config)) == 4
 end
+
+@testset "Biomass" begin
+    @test model_has_atpm_reaction(model)
+    wrong_model = convert(StandardModel, model)
+    remove_reaction!(wrong_model, "ATPM")
+    @test !model_has_atpm_reaction(wrong_model)
+
+    @test all(model_has_growth_atp_in_biomass(model))
+
+    @test "BIOMASS_Ecoli_core_w_GAM" in model_biomass_reactions(model)
```

Comment:
this can be an equality test

Reply:
nope, it returns a vector, so I could call `first` on it, but I would just type even more for the same effect :)